doc/release-notes.md

```diff
@@ -28,207 +26,77 @@ likely require a reindex.
-
-### Deprecations
-
-- **Deprecation of `-platform-user`**
-    - `-platform-user` is deprecated in favor of the whitelist feature.
-    - In releases **22.x** of Dash Core, it has been renamed to `-deprecated-platform-user`.
-    - It will be removed in version **23.x**.
+## New RPCs
 
-- **`-statsenabled` Deprecation**
-    - Now implied by the presence of `-statshost`.
-    - It will be removed in version **23.x**.
+- **`getislocks`**
+    - Retrieves the InstantSend lock data for the given transaction IDs (txids).
+      Returns the lock information in both human-friendly JSON format and binary hex-encoded zmq-compatible format.
 
-- **`-statshostname` Replacement**
-    - Deprecated and replaced with `-statssuffix` for better representation of the argument's purpose.
-    - **Behavior:** Behave identically to each other.
-    - It will be removed in version **23.x**.
+Updated RPCs
+------------
 
-- **`-statsns` Replacement**
-    - Deprecated and replaced with `-statsprefix` for better representation of the argument's purpose.
-    - **Behavior:** `-statsprefix` enforces the usage of a delimiter between the prefix and key.
-    - It will be removed in version **23.x**.
+- The top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and `descendantfees`
+  returned by RPCs `getmempoolentry`,`getrawmempool(verbose=true)`,
+  `getmempoolancestors(verbose=true)` and `getmempooldescendants(verbose=true)`
+  are deprecated and will be removed in the next major version (use
+  `-deprecated=fees` if needed in this version). The same fee fields can be accessed
+  through the `fees` object in the result. WARNING: deprecated
+  fields `ancestorfees` and `descendantfees` are denominated in sats, whereas all
+  fields in the `fees` object are denominated in DASH.
+- **`getbestchainlock` Changes**
+    - A new hex field has been added to the getbestchainlock RPC, which returns the ChainLock information in zmq-compatible, hex-encoded binary format.
+- `lockunspent` now optionally takes a third parameter, `persistent`, which
+  causes the lock to be written persistently to the wallet database. This
+  allows UTXOs to remain locked even after node restarts or crashes.
 
-## GUI Changes
+GUI changes
+-----------
 
-- **RPC Server Functionality Option**
-    - A new option has been added to the "Main" tab in "Options" that allows users to enable RPC server functionality.
+- UTXOs which are locked via the GUI are now stored persistently in the
+  wallet database, so are not lost on node shutdown or crash.
+- Improved GUI responsiveness for large wallets. (dash#6457)
 
-# v22.0.0 Change log
+# v22.1.0 Change log
 
 See detailed [set of changes][set-of-changes].
 
 # Credits
 
 Thanks to everyone who directly contributed to this release:
 
-- AJ ONeal
 - Kittywhiskers Van Gogh
 - Konstantin Akimov
-- Odysseas Gabrielides
 - PastaPastaPasta
 - UdjinM6
-- Vijaydasmp
```

Comment:
pls revert

https://github.com/dashpay/dash/issues?q=is%3Apr%20is%3Aclosed%20author%3Avijaydasmp%20milestone%3A22.1